### PR TITLE
Fabiane and Ian switched slots

### PR DESCRIPTION
**Pull request by boyarsky**
Confirmed with both Fabiane and Ian that the new slots are ok

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -409,29 +409,29 @@ <h3 style="color: red;">Abstract</h3>
...
+                    </p>
                   </div>
 
                 </div>
-                <div onclick="openAbstract('nardon')" title="Click for Abstract"><h2 class="text-white"><a href="#nardonCard" class="text-primary d-block mb-3" data-aos="fade-left" data-aos-delay="300">Data Engineering and Machine Learning Engineering for Java Developers</a></h2>
-                  <!--span class="popuptext" id="moraes">Abstract: <br/>They told you that using Java with containers were great, but they never mentioned that wouldn't be that easy, right? Luckily there are have some best practices that will rescue your application from failing. Come to this session and learn them.</span-->
+                <div onclick="openAbstract('darwin')" title="Click for Abstract"><h2 class="text-white"><a href="#darwinCard" class="text-primary d-block mb-3" data-aos="fade-right" data-aos-delay="300">Flutter is the new Java - or is it?</a></h2>
```
Comment (henri-tremblay) [inline schedule.html:428]: Why is it a fade-right instead of a fade-left? Same below, it switches. Why would a card fade on left instead of right?

Comment (henri-tremblay) [reply schedule.html:428]: Answering my question: I think it randomly currently fades left or right. Originally, it was probably ordered left, right, left, right. Doesn't matter.

**Comment by juneau001** [comment]
Thanks @boyarsky for the PR and thanks @henri-tremblay for reviewing.

File: schedule.html (hunk 1 above)
Comment (boyarsky) [reply schedule.html:428]: Thanks Henri. I didn't notice a pattern. Glad it turned out now to matter

**Pull request merged**